runner.sh

```diff
@@ -10,6 +10,7 @@ IFS=$'\n\t'
 #/   -F, --folder <folder>             Export folder name (default 'export').
 #/   -h, --help                        Display this help message.
 #/   --remove-page-suffix              Remove page suffix when possible
+#/   --on-changes                      Export drawio file only if he's newer than exported files
 #/                                     (in case of single page file)
```

Comment:
Rectify the lines order

```suggestion
#/   --remove-page-suffix              Remove page suffix when possible
#/                                     (in case of single page file)
#/   --on-changes                      Export drawio files only if it's newer than exported files
```